command/namespace_status.go

```diff
@@ -24,13 +24,26 @@ Usage: nomad namespace status [options] <namespace>
 
 General Options:
 
-  ` + generalOptionsUsage(usageOptsDefault|usageOptsNoNamespace)
+  ` + generalOptionsUsage(usageOptsDefault|usageOptsNoNamespace) + `
+
+  Status Specific Options:
+  
+	-json
+	  Output the latest namespace status information in a JSON format.
+  
+	-t
+	  Format and display namespace status information using a Go template.
+  `
```

Comment:
slight fix to the output formatting:

```suggestion
Status Specific Options:
  
  -json
    Output the latest namespace status information in a JSON format.
  
  -t
    Format and display namespace status information using a Go template.
`
```